cookbooks/bcpc/templates/default/sshd_config.erb

```diff
@@ -61,8 +61,13 @@ ChallengeResponseAuthentication no
 #KerberosTicketCleanup yes
 
 # GSSAPI options
+<% if node[:bcpc][:kerberos][:enable] == true %>
+GSSAPIAuthentication yes
+GSSAPICleanupCredentials yes
+<% else %>
 #GSSAPIAuthentication no
```

Comment:
The comments are part of stock sshd_config, I would rather keep it in its entirety just to make sure we keep whatever defaults are there